frontend/src/bundles/common/enums/theme-colors.ts

```diff
@@ -0,0 +1,46 @@
+const ThemeColors = {
```

Comment:
I don't think we need this object; it could be just inlined in components. It will be really hard to manage if there are a lot of components. Also, we probably need to change more styles, text colors, etc., because right now it looks so-so.

Reply:
Yes, we could need variants and it could grow a lot, right, I will remove it and add it inline to the components!